ncutixavier/ncutixavier.github.io
# Improved blog Page

### **What about this PR?**
This PR is about to review blog page check for responsive.

### **Description of Task to be completed**
1. List of articles
2. Open a single article and leave a comment
3. responsive to all devices

### **How should be manually tested?**
   1. After Cloning the repo
   2. open blog.html
   3. click on one article and leave a comment
   4. Reload the article to check for your comment
   5. Remember to check for responsiveness

### **Relevant pivot tracker**
[#xzLuBLNO](https://trello.com/c/xzLuBLNO/56-get-article), [#egPNuuvt](https://trello.com/c/egPNuuvt/27-responsive-style)

### **Other Information**
**Responsive**
![image](https://user-images.githubusercontent.com/51375428/91887174-b4babb80-ec8a-11ea-85c4-a3011a458b57.png)

**Comment**
![image](https://user-images.githubusercontent.com/51375428/91887341-f51a3980-ec8a-11ea-97de-a11f45911591.png)
